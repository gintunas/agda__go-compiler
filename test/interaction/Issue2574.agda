-- Andreas, 2017-05-17, issue #2574 reported by G. Allais

open import Issue2574Import        -- The imported module should be clickable.
open import Issue2574ImportBlank   -- The imported module should be clickable.
