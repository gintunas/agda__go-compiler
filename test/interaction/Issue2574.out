(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-highlight-add-annotations 'nil '(1 58 (comment) t) '(60 64 (keyword) t) '(65 71 (keyword) t) '(95 138 (comment) t) '(139 143 (keyword) t) '(144 150 (keyword) t) '(174 217 (comment) t))
(agda2-highlight-add-annotations 'nil '(72 87 (module) nil nil ("Issue2574Import.agda" . 1)) '(151 171 (module) nil nil ("Issue2574ImportBlank.agda" . 1)))
(agda2-highlight-add-annotations 'nil '(60 64 (keyword) t) '(65 71 (keyword) t) '(72 87 (module) nil nil ("Issue2574Import.agda" . 1)))
(agda2-highlight-add-annotations 'nil '(139 143 (keyword) t) '(144 150 (keyword) t) '(151 171 (module) nil nil ("Issue2574ImportBlank.agda" . 1)))
(agda2-highlight-add-annotations 'nil '(1 58 (comment) t) '(95 138 (comment) t) '(174 217 (comment) t))
(agda2-status-action "Checked")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
